Gopkg.toml

```diff
@@ -58,10 +58,6 @@
   name = "github.com/go-kit/kit"
   version = "0.7.0"
 
-[[constraint]]
```

Comment:
@powerslacker thanks for noting the inconsistency for logfmt import. However, let us not remove the constraint for this package. If you want to import the package as "github.com/go-logfmt/logfmt", can you please add source option to this constraint. You can look thrift constraint in the same file.

Reply:
Also looking at your code change, we internally found an issue with 

> version = "v0.1.0"

 can you please change the constraint version as below?

> version = "=v0.1.0"